gateway/src/samples/kotlin/EventListener.kt

```diff
@@ -19,8 +18,8 @@ suspend fun main(args: Array<String>) {
     val gateway = DefaultGateway {
         client = HttpClient(CIO) {
             install(WebSockets)
-            install(JsonFeature) {
-                serializer = KotlinxSerializer(Json)
+            install(ContentNegotiation) {
+                json()
```

Comment:
maybe here too, I'm not sure whether the previously passed `Json` was also custom

Reply:
It just passed in `Json.Companion`, which also was the default back then

Reply:
then we don't need it here 👍